src/Microsoft.Extensions.RazorViews.Sources/AttributeValue.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Extensions.RazorViews
```

Comment:
Oh nevermind I forgot we made these actually internal, so no need.

---
In reply to: [75359024](https://github.com/aspnet/Common/pull/139#discussion_r75359024) [](ancestors = 75359024,75024341)